src/lib.rs

```diff
@@ -1317,6 +1317,132 @@ pub unsafe trait FromBytes: FromZeroes {
         Ref::<&mut [u8], Self>::new_from_suffix(bytes).map(|(_, r)| r.into_mut())
     }
 
+    /// Interprets the given `bytes` as a `&[Self]` without copying.
+    ///
+    /// If `bytes.len() % size_of::<T>() != 0` or `bytes` is not aligned to
+    /// `align_of::<T>()`, this returns `None`.
+    ///
+    /// If you need to convert a specific number of slice elements, see
+    /// [`FromBytes::slice_from_prefix`] or [`FromBytes::slice_from_suffix`].
```

Comment:
Sorry, uber nit: Can you have these omit the trait name and then use markdown out-of-line link syntax? Given that these will generate intra-page doc links, I think the trait name prefix is redundant. I.e.:

```rust
/// ... [`slice_from_prefix`] ...
///
/// [`slice_from_prefix`]: FromBytes::slice_from_prefix
```